src/main/java/com/google/devtools/build/lib/exec/ExpandedSpawnLogContext.java

```diff
@@ -282,6 +283,11 @@ public void logSpawn(
     }
   }
 
+  @Override
+  public void logSymlinkAction(AbstractAction action) {
+    // The expanded log does not report symlink actions.
```

Comment:
I could represent these actions as synthetic spawns.

Reply:
I'm fine with not implementing it unless someone requests it.